src/decrypter-worker.js

```diff
@@ -6,43 +6,36 @@ import { createTransferableMessage } from './bin-utils';
  * Our web worker interface so that things can talk to aes-decrypter
  * that will be running in a web worker. the scope is passed to this by
  * webworkify.
- *
- * @param {Object} self
- *        the scope for the web worker
  */
-const DecrypterWorker = function(self) {
-  self.onmessage = function(event) {
-    const data = event.data;
-    const encrypted = new Uint8Array(
-      data.encrypted.bytes,
-      data.encrypted.byteOffset,
-      data.encrypted.byteLength
-    );
-    const key = new Uint32Array(
-      data.key.bytes,
-      data.key.byteOffset,
-      data.key.byteLength / 4
-    );
-    const iv = new Uint32Array(
-      data.iv.bytes,
-      data.iv.byteOffset,
-      data.iv.byteLength / 4
-    );
+self.onmessage = function(event) {
```

Comment:
We no longer wrap in a function here, instead the code looks exactly as it would running on a web worker, inside of the main script body.